### ISSUE TITLE
[Bug] Crash Loop Bug

### ISSUE THREAD

**Issue by RelativeSpade** [MeteorDevelopment/meteor-client#1779]
### Describe the bug

The Bug crashes your pc if you run out of exp bottles while on a server, I had 3 and a half stacks of exp bottles and once I ran out of them the computer crashed.

### Steps to reproduce

```bash
Have 3 and a half stacks of exp bottles in the last 4 slots of Hotbar, full netherite in other slots, turn on exp bottle thrower doesn't turn it off, and once exp bottles crash.
```

### Meteor Version

0.4.7 #1362

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by MineGame159** [comment]
Send the crash report. @RelativeSpade

**Comment by xsolo9** [comment]
I have the exact same issue. Here's the crash report since op has gone AWOL:
[crash-2021-12-26_15.13.01-client.txt](https://github.com/MeteorDevelopment/meteor-client/files/7778011/crash-2021-12-26_15.13.01-client.txt)
.

**Comment by MineGame159** [comment]
Fixed in 96604b46d0514baba65e795fe0d349234ed9d1aa.
